cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -251,16 +291,17 @@ func init() {
 	MigrateApplyCmd.Flags().SortFlags = false
 	cobra.CheckErr(MigrateApplyCmd.MarkFlagRequired(migrateFlagURL))
 	MigrateApplyCmd.MarkFlagsMutuallyExclusive(migrateApplyFromVersion, migrateApplyBaselineVersion)
-	cobra.CheckErr(MigrateApplyCmd.Flags().MarkHidden(migrateFlagDirFormat))
-	cobra.CheckErr(MigrateApplyCmd.Flags().MarkHidden(migrateFlagForce))
-	cobra.CheckErr(MigrateApplyCmd.Flags().MarkHidden(migrateFlagSchema))
 	// Diff flags.
```

Comment:
Why? Also, can we remove the `--force` one?

Reply:
Because this marks the flags hidden for ALL commands and not only the apply command. Cobra is a little weird here.

Keeping this means having the flags not show in any help command.

Reply:
> Why? Also, can we remove the `--force` one?

Yes, but not in this PR.